externs/olgmx.js

```diff
@@ -34,12 +34,13 @@ olgmx.OLGoogleMapsOptions.prototype.mapIconOptions;
 
 
 /**
- * Whether the library should watch vector layers and let them be rendered
- * by Google Maps with the latter is activated or not.  Defaults to `true`.
- * @type {boolean|undefined}
+ * For each layer type, a boolean indicating whether the library should watch
+ * and let layers of that type them be rendered by Google Maps or not.
+ * Defaults to `true` for each option.
+ * @type {olgmx.herald.WatchOptions|undefined}
  * @api
  */
-olgmx.OLGoogleMapsOptions.prototype.watchVector;
+olgmx.OLGoogleMapsOptions.prototype.watchOptions;
```

Comment:
It's implicit that those are options... I suppose we could name this `watch` only.  The definition itself could keep its name.